packages/engine-rn-web/src/sdks/sdk-webos/index.ts

```diff
@@ -85,36 +95,36 @@ export const runWebOS = async (c) => {
     }
 };
 
-export const buildWebOSProject = async (c) => {
+export const buildWebOSProject = async (c: RnvContext) => {
     logTask('buildWebOSProject');
 
     await buildCoreWebpackProject(c);
 
     if (!c.program.hosted) {
         const tDir = getPlatformProjectDir(c);
-        const tOut = path.join(getPlatformBuildDir(c), 'output');
+        const tOut = path.join(getPlatformBuildDir(c)!, 'output');
```

Comment:
this feels a bit weird, no `!` on the one above and the same function called on the next line, instead of using tDir

Reply:
not weird. getPlatformProjectDir(c) returns string | undefined (that's correct) . ! is used only as temp workaround for path (which requires string and no udefined) to keep same func behaviour until logic refactor arrives

Reply:
what I meant was 
```
const tDir = getPlatformProjectDir(c)!;
const tOut = path.join(tDir, 'output');
```